helm-org.el

```diff
@@ -137,6 +147,16 @@ NOTE: This will be slow on large org buffers."
           :buffer "*helm org inbuffer*")))
 
 ;;;###autoload
+(defun helm-org-parent-headings ()
+  "Preconfigured helm for org headings that are parents of the current heading."
+  (interactive)
+  (helm :sources (helm-source-org-headings-for-files
+                  (list (current-buffer))
+                  1 8 t)
```

Comment:
alphapapa <notifications@github.com> writes:

> In helm-org.el:
>
>> @@ -137,6 +147,16 @@ NOTE: This will be slow on large org buffers."
>>            :buffer "*helm org inbuffer*")))
>>  
>>  ;;;###autoload
>> +(defun helm-org-parent-headings ()
>> +  "Preconfigured helm for org headings that are parents of the current heading."
>> +  (interactive)
>> +  (helm :sources (helm-source-org-headings-for-files
>> +                  (list (current-buffer))
>> +                  1 8 t)
>
> I hesitated to hard-code 1 8 here, but I don't know how else to do it,
> unless the helm-source-org-headings-for-files function were to have
> its arguments rearranged, which seems like a bad idea. Of course, I am
> an elisp novice, so maybe there is a better way to do this. :)

Perhaps use (parent t) in helm-source-org-headings-for-files and then
you don't have to specify 1, 8 and parent in helm-org-parent-headings:

    (helm :sources (helm-source-org-headings-for-files
                    (list (current-buffer)))

-- 
Thierry
https://emacs-helm.github.io/helm/